Document secureCookies flag and relationship with sameSite flag (#620)
blitz-js/blitzjs.com: app/pages/docs/session-management.mdx

type SessionConfig = {
  cookiePrefix?: string /* Default: 'blitz' */
  sessionExpiryMinutes?: number /* Default: 30 days */
  sameSite?: "strict" | "lax" | "none" /* Default: 'lax'. 

>     If setting this to 'none', you must also set secureCookies: true. 

Reviewer: ```suggestion
    If you set this to 'none', you also have to set `secureCookies: true`. 
```
